### ISSUE TITLE
Add option to disable automatic updates check & install

### ISSUE THREAD

**Issue by i0xHeX** [roldanjr/pomatez#320]
Currently, if there is a newer version, it doesn't ask you what you want to do with it, whether to install it or discard it. It always installs a new version, and that's kind of annoying if you can't handle it. This even happens when you run a portable version, so it will install a new version every time you run it.

Adding options to disable the automatic update check and automatic install (and making the last one disabled by default) is a good solution.

**Comment by sekwah41** [comment]
I'll review the update logic when I get some time. I didn't add it but I could possibly alter the logic just so it has a little banner that pops up at the top or bottom that you can dismiss or a modal over the timer that says an update is available.

I'll make it add an update and a changes button that links to the Release.

If its dismissed ill also make the update available to click from the settings menu. As ill make the dismiss logic not tell you again until a new version is available after the one you dismissed.

It would also probably be a good idea to add a settings toggle to disable checking for updates. Especially if the portable version you requested @i0xHeX gets added.

**Comment by unfa** [comment]
Yes, I have a similar problem where Pomatez opens a website automatically at every system start. What'd be enough is to have a tiny yellow circle with an arrow pointing up show in the UI. After clicking it could say "new update available! and give me an option to ignore to dismiss the notification or install).  Right now the tool gets in the way of my work :)

**Comment by sekwah41** [comment]
Can you give the new Tauri version a try @unfa? I am looking for testers anyway and I may look to transfer the new update logic to electron too if we continue that version. (though my focus atm is the rust version to get a mobile version out)
